graphql-demo-nextjs/src/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { State } from './types'
+
+export const createState = (): State => ({
+  todos: [
+    {
+      id: '1',
+      title: 'Deploy to GraphCDN',
+      createdAt: new Date().toISOString(),
+      user: '1',
+    },
+  ],
+  users: [{ id: '1', name: 'GraphCDN User' }],
+})
+
+export const allowedOrigins = process.env.ALLOWED_ORIGINS?.split(',') || ['*']
```

Comment:
constants we should prefer `UPPER_SNAKE_CASE`

```suggestion
export const ALLOWED_ORIGINS = process.env.ALLOWED_ORIGINS?.split(',') || ['*']
```